arithmetization/src/main/java/net/consensys/linea/zktracer/ColumnHeader.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package net.consensys.linea.zktracer;
+
+public record ColumnHeader(String name, int bytesPerElement, int length) {
+  public int dataSize() {
+    return this.length() * this.bytesPerElement();
+  }
+
+  public int headerSize() {
+    return 2
```

Comment:
What is the `2` in the formula ?

Reply:
2 bytes for the length of the string (`// i16: name size`) but spotless puts the comment on another line.